misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickStepButton.jsx

```diff
@@ -120,7 +120,7 @@ PickStepButton.propTypes = {
   qtyPicked: PropTypes.number,
   qtyToPick: PropTypes.number.isRequired,
   stepState: PropTypes.object,
-  pickFromAlternatives: PropTypes.array,
+  pickFromAlternatives: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
```

Comment:
why one of ? isn't always the same type?

Reply:
yes should be always array